src/tox/config/source/discover.py

```diff
@@ -26,6 +26,17 @@ def discover_source(config_file: Path | None, root_dir: Path | None) -> Source:
         src = _locate_source()
         if src is None:
             src = _create_default_source(root_dir)
+    elif config_file.is_dir():
+        src = None
+        for src_type in SOURCE_TYPES:
+            candidate: Path = (config_file / src_type.FILENAME).resolve()
```

Comment:
Do we really need the resolve? I think can do without 🤔

Reply:
Not really the code from `_create_default_source` is not compatible with our needs. We would need to do extra changes in it to make it reusable for the "folder" case.

Reply:
I wasn't talking about reusing that method here, but about not using resolve in this line.

Reply:
True, I removed it. I added it while testing when something did not work right and I wanted to see how it goes.